packages/serializer-md/src/deserializer/utils/deserializeMd.spec.tsx

```diff
@@ -294,7 +294,7 @@ describe('deserializeMd', () => {
       <fragment>
         <hp>
           <htext>Line 1</htext>
-          <htext>{'\n'}</htext>
+          <htext>{'<br>'}</htext>
```

Comment:
Expecting this to return a text node containing the string `'<br>'` doesn't seem right to me

Reply:
Updated